### ISSUE TITLE
will get better results when we set 'self.clip_denoised = False' ?

### ISSUE THREAD

**Issue by JunyaoHu** [CompVis/latent-diffusion#239]
Dear author, thanks for your nice work, I want to know the effect of parameter ‘self.clip_denoised’, in your class DDPM you make 'self.clip_denoised = True' but in LatentDiffusion you force 'self.clip_denoised = False'.

https://github.com/CompVis/latent-diffusion/blob/2b46bcb98c8e8fdb250cb8ff2e20874f3ccdd768/ldm/models/diffusion/ddpm.py#L60

https://github.com/CompVis/latent-diffusion/blob/2b46bcb98c8e8fdb250cb8ff2e20874f3ccdd768/ldm/models/diffusion/ddpm.py#L463

In my practice, if it is False, it will make training harder, because the result x0 will bigger and bigger in an edited video diffusion model based on LDM. X0 is difficult to converge.

**Comment by TuanDTr** [comment]
Hi @JunyaoHu, I wonder if you have additional insights for this. I think that for LDM since the latents do not have a range, forcing it to be between -1 and 1 during sampling could be counterintuitive. What are your thoughts on this?

**Comment by JunyaoHu** [comment]
> Hi @JunyaoHu, I wonder if you have additional insights for this. I think that for LDM since the latents do not have a range, forcing it to be between -1 and 1 during sampling could be counterintuitive. What are your thoughts on this?

Thank you for your reply. I've reconsidered the question and I agree with you. I didn't notice at first that the coded values had no range. 

Empirically, I found that most of the values encoded by LDM are in the middle of [-5,5], so I scaled them to approximately [-1,1] using the scale_factor parameter. 

I use this method, coupled with clipping values, which may sometimes work(a little faster training), but it doesn't really make much sense.
